### PR TITLE
footer.html: Create footer

### PR DESCRIPTION
**Pull request by hemangsk**
Closes https://github.com/coala/landing-frontend/issues/18
screen shot 2017-01-19 at 11 55 00 pm

### CODE REVIEW

File: includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                    <li><a class="grey-text text-lighten-3" href="http://blog.coala.io/">Blog</a></li>
+                    <li><a class="grey-text text-lighten-3" href="https://gratipay.com/coala/">Donate</a></li>
+                </ul>
+            </div>
+            <div class="col-md-9">
+                <div class="row">
+                <div class="col-md-4">
+                    <div class="footer-small-title">Sponsored By</div>
+                    <div class="footer-sponsor-img center">
+                    <a href="http://appveyor.com"><img height=30 src="resources/images/av.png"/></a>
```
Comment (sils) [inline includes/footer.html]: appveyor sponsors server resources just as digitalocean so let's put them into the same category

**Comment by sils** [comment]
I'd put the twitter/chat buttons above the sponsorship, also the alignment seems a bit chaotic still

**Comment by hemangsk** [comment]
on it

**Comment by hemangsk** [comment]
wrt 	3b02c6f


screen shot 2017-01-20 at 4 50 51 am

**Pull request merged**